packages/app/src/game/module/board/overlays/quickChat/quickChatButtonArray.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { QuickChatButton } from "./quickChatButton";
+import { QuickChatOption } from "@creature-chess/models";
+
+
+const QuickChatButtonArray: React.FunctionComponent = () => {
+	// to have different options for different overlays etc., pass the Options enum as a parameter to this component
+	// and map to array as below.
+	const quickChatArray = Object.values(QuickChatOption).map(option => option);
```

Comment:
I'm not sure what the `.map(option => option)` is for here but I don't think it's necessary